$OpenBSD: patch-src_lib_asiolink_io_service_h,v 1.1 2018/01/31 22:29:09 rsadowski Exp $

fix build with boost 1.66
- use io_context instead of io_service
commit 17ed090beb6cefa78e29a95c645eb6da4edf8

Index: src/lib/asiolink/io_service.h
--- src/lib/asiolink/io_service.h.orig
+++ src/lib/asiolink/io_service.h
@@ -11,7 +11,12 @@
 
 namespace boost {
 namespace asio {
+#if BOOST_VERSION < 106600
     class io_service;
+#else
+    class io_context;
+    typedef io_context io_service;
+#endif
 }
 }
 
